examples/gno.land/r/gnoland/pages/page_contribute.gno

```diff
@@ -0,0 +1,91 @@
+
+If you are interested in contributing to gno.land, you can jump on in on our [GitHub monorepo](https://github.com/gnolang/gno/blob/master/CONTRIBUTING.md) - where most development happens. Additionally, you can look out to help on specific issues labeled as bounties. All contributions will then concur to form your profile for Game of Realms. In the following sections, we delve into these initiatives in a bit more detail.
+
+## Gno Bounties
+
+The Gno bounty program is a good way to familiarize yourself with the project, and how Gno code works. We will maintain open and rewardable bounties in the gnolang/gno repository, and you can search all available bounties by using the ["bounty" label](https://github.com/gnolang/gno/labels/bounty).
+
+Recommendations on participating in the gno.land Bounty Program:
+
+- Identify the bounty you want to work on, and first start working in your local environment.
+- Once you have spent time on the code related to the bounty, we recommend submitting a 'draft' PR as soon as possible.
+    - The draft PR doesn't indicate that the bounty has been assigned to you, others are free to work on other draft PRs for the bounty.
+    - After submitting the 'draft' PR, continue working until you are ready to mark the PR as "ready for review".
+    - The core team will review the bounty PR submission, and determine if it qualifies for the bounty reward.
+- Ask for clarification early if an element on the DoD or implementation design is unclear.
+    - Aside from posting the issue as a bounty, keeping regular updates with the core team on the bounty being undertaken is key to being on the right track.
+    - The Definition of Done requires adhesion to the [contributing guidelines](https://github.com/gnolang/gno/blob/master/CONTRIBUTING.md); additionally, it is expected that any newly added code or functionality is properly documented, tested and covered, at least in 80% of added code.
+    - You're welcome to propose additional features and work on an issue should you envision a plausible expansion or change in scope. The core team may assign a bounty to the additional work, or change the bounty with respect to the changed scope.
+
+You may make your submission at any time; however we invite you to publish your draft PR very early in the development process. This will make your work public, so you can easily get help by the core team and other community members. Additionally, your work can be continued by other people should you get stuck or no longer be willing to work on the bounty. Likewise, you can continue the abandoned or stuck work that someone else worked on. Don't fear your work being "stolen": if a submission is the result of multiple people's efforts, we will look to split the bounty in a way that is fair and recognises each participant in creating the final outcome.
```

Comment:
this paragraph is important; we should mabye be more explicit on how we may adjust the things, like:
- if your PR addresses 2 bounties
- if you do 80% of the job, and we merge the PR and someone else later do 20%
- if you do a first PR, and someone forks your PR or get inspired by it, and we merge the new PR
- if you do the PR and beyond (make some useful yak shaving)

i think we can keep the expected compensation mechanism but we need to explain why sometimes we could give less or more; and just specify that we'llexplain our reasoning

Reply:
I wouldn't explicitly mention a PR addressing two bounties. We should have people look to make small PRs.

Reply:
I added some examples, lmk